clarity_ext/domain/aliquot.py

```diff
@@ -21,11 +22,7 @@ def __init__(self, api_resource, is_input, id=None, samples=None, name=None,
                                       name=name,
                                       udf_map=udf_map,
                                       is_input=is_input,
-                                      mapper=mapper)
-        # NOTE: This is a quick fix for extremely slow loading of large pools
-        if samples:
-            self._samples_require_initializing = not isinstance(samples[0], Sample)
-        self._samples = samples
+                                      mapper=mapper,)
```

Comment:
Is that extra comma something the IDE is adding?

Reply:
I have a feeling that's me adding and removing lines :/